### ISSUE TITLE
Issue about the network.py

### ISSUE THREAD

**Issue by accuracy-maker** [venite-xjc/SuperTrack#2]
Thanks for your great work!  When I look through the code, I found that :
`       world_state_dict = torch.load('/root/SuperTrack/model1/epoch_119_world.pth')
        policy_state_dict = torch.load('/root/SuperTrack/model1/epoch_119_policy.pth')
        self.world.load_state_dict(world_state_dict)
        self.policy.load_state_dict(policy_state_dict)`
It seems that you use the trained model. But I don't have the files. So could I delete these lines to run the code?

**Comment by venite-xjc** [comment]
Of course, just delete these lines. But training both networks with large window sizes is difficult, I suggest you start with small window sizes to get initial weights, then increase the size gradually.

**Comment by accuracy-maker** [comment]
ok, thx very much

**Comment by accuracy-maker** [comment]
when I train this network, the loss increases dramatically after specific epoch and becomes NAN. Did you encouter this situation?
